### PR TITLE
Update deprecated mlab connection guide for the new MongoDB Atlas

### LINKED ISSUE #35587: MongoDB and Mongoose - Install and Set Up Mongoose
**Describe your problem and how to reproduce it:**
mLab is no longer accepting new users, cannot get mongoose to connect in order to complete challenges.  Directions are now obsolete - need update.

**Add a Link to the page with the problem:**
https://i.imgur.com/JPAgbHj.png

**Tell us about your browser and operating system:**
* Browser Name: chrome
* Browser Version: 72.0.3626.121
* Operating System: MAC OS 10.14.3

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
Screen Shot 2019-03-13 at 12 14 18 PM

### PR DESCRIPTION
**Pull request by Dereje1**
- [X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] None of my changes are plagiarized from another source without proper attribution.
- [X] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #35587

### More detailed explanation
mLab was recently acquired by MongoDB, Inc. As of March 2019, new mlab accounts can not be created and therefore the instructions provided [here](https://learn.freecodecamp.org/apis-and-microservices/mongodb-and-mongoose) are no longer valid. 
Following the updated instructions in this PR will enable the creation of a new MonogDB Atlas connection URI that is necessary to continue with the lessons.

### CODE REVIEW

**Review by RandellDawson — APPROVED** [review]
I read through the instructions and was able to set up a new account without any issues.  

@raisedadead I also think these instructions should be updated ASAP on the legacy production site, because this is currently stopping campers in their progress of the current curriculum in production.

**Comment by raisedadead** [comment]
I'll defer to @scissorsneedfoodtoo for making the changes to the production learn.

**Comment by raisedadead** [comment]
@RandellDawson Please merge this PR is this is ready.

File: client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md [hunk 1]
```diff
@@ -13,63 +13,68 @@ MongoDB is a database that stores data records (documents) for use by an applica
...
-- Select the *Sandbox* plan type, which is the only one with no cost, and press **Continue**.
-- Select a region for your Sandbox, from the available list, and press **Continue**.
-- Input a name for your database. This name will be used in the URI for your database. After that, press **Continue**.
+### Create a new admin user on the database.
 
-A summary of all your choices should appear, allowing you to change any information provided in the previous steps. Press **Submit Order** to confirm the information.
+- You should be able to see the green **Get Started** button on the bottom left of your screen, you can click this button to see at which step of the process your are in, if you click on it now, you can see the next step is to **Create your first database user**, go ahead and click on that step.
+    - Follow the instructions by clicking on the `Security` tab.
+    - Click on the green **ADD NEW USER** button.
+    - Enter a user name and password and then select **Atlas Admin** under user privileges, remember to store your username and password somewhere safe.
```
Comment (ojeytonwilliams) [inline client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md]:
```suggestion
    - Enter a user name and password and then select **Read and write to any database** under user privileges, remember to store your username and password somewhere safe.
```
No need for admin privileges here, the projects aren't going to need them.

File: client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md [hunk 2]
```diff
@@ -13,63 +13,68 @@ MongoDB is a database that stores data records (documents) for use by an applica
...
-- Click the **Add database user** button.
-- Input an username and password for your administrator. Do not mark it as read-only or you will not be able to add any information to the database with this user.
+- Clicking on the green **Get Started** button in the bottom left of your screen should now show you the next step, **Connect to your cluster**, click on it.
+    - Follow the instructions by clicking on the `Connect` button in the `Sandbox` section.
+    - In the pop-up modal, click on **Connect Your Application**, a connection string will be displayed, you can copy that connection string by clicking on the `copy` button.
+    - This will be the final URI that you will use to connect to your db, it will look something like this `mongodb+srv://<user>:<password>@<cluster#-dbname>.mongodb.net/test?retryWrites=true`, notice that the `user` and `cluster#-dbname` fields are already filled out for you, all you would need to replace is the `password` field with the one that you created in the previous step. 
 
-### Get the mLab URI
+- That's it! you now have the URI you will add to your application to connect to your database. Keep this URI safe somewhere, so you can use it later!
```
Comment (ojeytonwilliams) [inline client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md]:
```suggestion
- That's it! You now have the URI you will add to your application to connect to your database. Keep this URI safe somewhere, so you can use it later!
```

**Comment by scissorsneedfoodtoo** [comment]
@RandellDawson, yes, let's update the instructions here since this could block campers working through the curriculum now. I also went through the instructions and made an account.

Thank you for updating these intructions @Dereje1! Everything LGTM.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Thanks for this, it's very detailed!  Comments below.

File: client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md (hunk 2 above)
Comment (scissorsneedfoodtoo) [reply client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md]: Totally missed that one! Great catch there @ojeytonwilliams.

File: client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md (hunk 1 above)
Comment (Dereje1) [reply client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md]: Applied the changes per gitter conversation.

File: client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md (hunk 2 above)
Comment (Dereje1) [reply client/src/pages/learn/apis-and-microservices/mongodb-and-mongoose/index.md]: Applied changes.

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
@Dereje1 Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Comment by ojeytonwilliams** [comment]
@scissorsneedfoodtoo Everything should be ready to go into production.

**Comment by scissorsneedfoodtoo** [comment]
Thank you for all of your hard work @Dereje1! Looking forward to seeing this on the live site. And thank you for the heads up, @ojeytonwilliams. We'll get this out soon.

**Comment by RandellDawson** [comment]
@scissorsneedfoodtoo How long before we see these instructions appear on the production site?

**Comment by scissorsneedfoodtoo** [comment]
@RandellDawson, I can't say for sure, but hopefully within the next week.

**Comment by RandellDawson** [comment]
@scissorsneedfoodtoo I was thinking for critical updates (like this one), is there not a way to update the legacy production?

**Pull request merged**